From 756c1673881ae104e20305fadad41cfea5f04608 Mon Sep 17 00:00:00 2001
From: Mike Perry <mikeperry-git@torproject.org>
Date: Thu, 1 Jan 1970 00:00:00 +0000
Subject: [PATCH 29/29] Disable library timestamping.

This timestamp prevents deterministic builds, and is set in a way that
libfaketime can't correct.
---
 nsprpub/config/now.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/nsprpub/config/now.c b/nsprpub/config/now.c
index 165b924..5e07a72 100644
--- a/nsprpub/config/now.c
+++ b/nsprpub/config/now.c
@@ -14,7 +14,7 @@
 #error "Architecture not supported"
 #endif
 
-
+#define OMIT_LIB_BUILD_TIME
 int main(int argc, char **argv)
 {
 #if defined(OMIT_LIB_BUILD_TIME)
-- 
1.7.9.5

